mongo/integration/unified/client_entity_test.go

```diff
@@ -25,11 +25,12 @@ import (
 type ClientEntity struct {
 	*mongo.Client
 
-	recordEvents    atomic.Value
-	started         []*event.CommandStartedEvent
-	succeeded       []*event.CommandSucceededEvent
-	failed          []*event.CommandFailedEvent
-	ignoredCommands map[string]struct{}
+	recordEvents     atomic.Value
+	started          []*event.CommandStartedEvent
+	succeeded        []*event.CommandSucceededEvent
+	failed           []*event.CommandFailedEvent
+	ignoredCommands  map[string]struct{}
+	serverAPIOptions *options.ServerAPIOptions
```

Comment:
Do we need to store this here or can we just use it to construct the underlying `mongo.Client`? I don't see any uses after Client construction, so I'd suggest removing it for now.

Reply:
Great point, removed!